client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/entities/SchemaString.java

```diff
@@ -25,6 +25,7 @@
 public class SchemaString {
```

Comment:
Add `@JsonInclude(JsonInclude.Include.NON_NULL)` before `SchemaString` so that `hwm` is not sent if it is NULL (for backward compatibility).  See `RegisterSchemaRequest` in `pluggable-sr-client` branch.